reapy/tools/__init__.py

```diff
@@ -1,4 +1,4 @@
 """Define tools such as ``inside_reaper`` and custom json module."""
 
 import reapy
-from .inside_reaper import inside_reaper, dist_api_is_enabled
+from .inside_reaper_ import inside_reaper, dist_api_is_enabled
```

Comment:
Wouldn't it be better to rename the module `_inside_reaper` instead? Leading underscores for internal use names is a very usual Python convention. Conversely, I have almost never seen trailing underscores, except in some packages where they have a special meaning for mutable objects.